### PR TITLE
Upgrade of Mobile Portfolio App for Desktop Responsivness

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project, I enhanced my Mobile Portfolio app to be responsive to desktop screens.
I used:

- [ ] HTML
- [ ] CSS
- [ ] Linters
- [ ] I also used Grid and Flexbox to display contents on the page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -25,424 +17,1056 @@ a {
...
-  height: 40px;
-  padding: 10px 12px 10px 12px;
-}
+  .multi-stories {
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    font-weight: 400;
+    text-align: left;
+    margin-left: 10px;
+  }
```
Comment (AlphaNtihinduka) [inline index.css:242]: - [ ] You may need to ensure that your portifolio matches the Figma design as much as possible. If you are keen at this heading, you can realize that it crosses into the above line. For this reason, I suggest that you consider having the heading below the line and at the same level as the image ☺️.

Your design 😎 
image

Figma design 😎 
image

File: index.html [hunk 2]
```diff
@@ -37,24 +45,65 @@ <h2>I'm a Software Developer</h2>
...
               <li><a class="second-list" href="#">CSS</a></li>
               <li><a class="third-list" href="#">Javascript</a></li>
             </ul>
             <button>See Project</button>
           </article>
+        
+        <div class="print-container">
+          <article class="print-data data-0">
+            <button>See Project</button>
+          </article>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] Please try as much as possible to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=E5AVHIf7a3eVNCi2-0). There should be a spacing on the left of the cards to ensure that your portifolio is more user-friendly ☺️.
- [ ] In addition, the spacing between the cards should be reduced to match the Figma design ☺️.
Your design 😎 
image

Figma design 😎 
image

File: index.html [hunk 3]
```diff
@@ -37,24 +45,65 @@ <h2>I'm a Software Developer</h2>
 
         <article class="extra">
           <section class="works-centered">
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] I suggest to consider aligning the section to the left such that it is at the same level as the heading and the cards. This is to ensure that we stick the original Figma design ☺️.
- [ ] In addition, `project-list` should not overlap into the `p`. This makes your design messy and less user-friendly ☺️. Please fix this to make sure that your portifolio is professional 👍 
- [ ] Finally on this section, I kindly suggest to make sure that the `p` text is responsive in all the desktop screen sizes 👍 

Your design 😎 
image

Figma design 😎 
image

File: index.css [hunk 4]
```diff
@@ -25,424 +17,1056 @@ a {
...
+    position: fixed;
+    top: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #3c3a39;
+    margin-left: 0;
+  }
```
Comment (AlphaNtihinduka) [inline index.css:101]: - [ ] I have noticed that there is a spacing in the left side of the `header`. Kindly make sure that this spacing is removed to ensure that your portifolio has good UI and hence professional ☺️ 

Your design 😎 
image

Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @anyars-encarta 🙋‍♂️
Good job working on desktop version of your portifolio. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Used media queries ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comments ☺️
- Finally, you should make sure that your portifolio doesn't have the horizontal scroll. The experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. Therefore I suggest that you ensure that your portifolio is horizontal scroll free.

https://github.com/anyars-encarta/Porfolio/assets/99394374/ae3cc17f-2ff9-485a-957d-c231fea68e5d



## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: index.css [hunk 5]
```diff
@@ -25,424 +17,1060 @@ a {
...
 }
 
-.attach-icon {
-  display: flex;
-  align-items: center;
-  justify-content: center;
+/* ***************************************************************************************************************** */
+@media (min-width: 768px) {
+  main {
```
Comment (joskalenda) [inline index.css]: - [ ] As mentioned in the previous CR, your application is not responsive on this screen size YET, I think you should have started the desktop design from this, and the value above it should match or be easily modified, see the record below,

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @anyars-encarta , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Green linters
- [ ]... ✅ Great readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @anyars-encarta,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Fixed suggested changes.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by anyars-encarta** [comment]
Thanks @AdnanOlarmmi . I am very grateful. I would very much love to learn from you. I am following you on GitHub from here.

**Pull request merged**